web_frontend/osmaxx/countries/utils.py

```diff
@@ -0,0 +1,75 @@
+import os
+
+from django.contrib.gis.geos import MultiPolygon, Polygon, GEOSGeometry
+
+from osmaxx.countries._settings import POLYFILE_LOCATION
+
+POLYFILE_ENDING = '.poly'
+
+
+def get_polyfile_name_to_file_mapping():
+    polyfile_mapping = {}
+    for possible_polyfile in os.listdir(POLYFILE_LOCATION):
+        if possible_polyfile.endswith(POLYFILE_ENDING):
+            name = possible_polyfile.split(POLYFILE_ENDING)[0]
```

Comment:
Didn't think of that. But actually, this seems problematic, because the filename can be virtually any string. 
I think, `possible_polyfile.strip(POLYFILE_ENDING)` would make more sense, right?